theme/templates/resource-landing-page/manage-access.html

```diff
@@ -198,7 +198,7 @@ <h3 class="panel-title">Sharing status</h3>
                                     <div class="checkbox">
                                         <label>
                                             <input type="checkbox" v-model="resAccess.isShareable" :disabled="isProcessingShareable"
-                                                   @change="setShareable(resAccess.isShareable ? 'make_not_shareable' : 'make_shareable')">
+                                                   @change="setShareable(resAccess.isShareable ? 'make_shareable' : 'make_not_shareable')">
```

Comment:
@Maurier - Found a bug with toggling the sharing checkbox.  Basically it just sets the opposite value of what it should, so there is no way for someone to toggle the shareable status.  I believe this was introduced with the vue facelift.